specification/web/resource-manager/Microsoft.Web/2016-03-01/ResourceProvider.json

```diff
@@ -967,6 +1016,35 @@
           "type": "string"
         }
       }
+    },
+    "VnetParameters": {
+      "description": "",
```

Comment:
Could you provide a 'description' for the type? The description is used to generate API comments and documentation so it is useful for our customers.